test-fuzz/tests/integration/ci.rs

```diff
@@ -232,7 +232,16 @@ fn unmaintained() {
         .success();
 }
 
+#[allow(deprecated)]
+#[test]
+fn actionlint() {
+    let home = env::home_dir().unwrap();
+    Command::new(home.join("go/bin/actionlint"))
```

Comment:
```suggestion
    Command::new(home.join("go/bin/actionlint"))
        .env("SHELLCHECK_OPTS", "--exclude=SC2002")
```
Sorry for not being explicit. I meant to use `SHELLCHECK_OPTS` here. I think this should work.